Develop

Alexandre-A11 opened the pull request.

DanielCamposDevX reviewed (changes requested):
Remember to place deploy link on the pull request conversation.
Footer with more space on the bottom than it needs.
Phone icon on the header should show cellphone number on hover and should be a link to start a call

Alexandre-A11 commented:
Hi @DanielCamposDevX,

1. Apologies for that — here’s the deploy link: [DEMO LINK](https://alexandre-a11.github.io/layout_landing-page/).

2. About the footer: could you clarify? I followed the Figma design and the measurements are almost the same, so I’m not sure why it appears to have excess spacing.

3. I’ve fixed the phone icon and added an animation as well.

Awaiting your feedback!
